src/core/electrostatics_magnetostatics/coulomb.hpp

```diff
@@ -60,8 +60,7 @@ struct Coulomb_parameters {
 extern Coulomb_parameters coulomb;
 
 namespace Coulomb {
-void calc_pressure_long_range(Observable_stat &virials,
-                              Observable_stat &p_tensor,
+void calc_pressure_long_range(Observable_stat &p_tensor,
```

Comment:
You should be able to remove the include for `Observable_stat` now

Reply:
ah right, forgot about the pressure observable...

Reply:
This should work the same way as energy, and then there should be no mention of Obsstat anymore...